src/main/java/frc/robot/subsystems/LauncherShoulder.java

```diff
@@ -80,6 +80,14 @@ public void periodic() {
         SmartDashboard.putNumber("Shoulder Absolute" , encoder.getAngle().getDegrees());
         SmartDashboard.putNumber("Shoulder Ground Relative" , Math.toDegrees(getShoulderAngleRadians()));
         SmartDashboard.putBoolean("Launcher at goal", atGoal());
+
```

Comment:
This work needs done before we set output based on the goal.   At the top of the method.

I have also realized we should cap output in the set control, that is not allow positive control when setpoint is above max nor negative control values if setpoint is below min, to also guard against pid error.

Reply:
I have made changes, we can review it at the meeting